server_addon/maya/server/settings/publishers.py

```diff
@@ -387,13 +393,17 @@ class ExtractAlembicModel(BaseSettingsModel):
     dataFormat: str = SettingsField(
         enum_resolver=extract_alembic_data_format_enum, title="Data Format"
     )
-    melPerFrameCallback: str = SettingsField(title="melPerFrameCallback")
-    melPostFrameCallback: str = SettingsField(title="melPostFrameCallback")
+    melPerFrameCallback: str = SettingsField(title="Mel Per Frame Callback")
+    melPostFrameCallback: str = SettingsField(title="Mel Post Frame Callback")
     preRollStartFrame: int = SettingsField(title="Pre Roll Start Frame")
-    pythonPerFrameCallback: str = SettingsField(title="pythonPerFrameCallback")
-    pythonPostJobCallback: str = SettingsField(title="pythonPostJobCallback")
-    userAttr: str = SettingsField(title="userAttr")
-    userAttrPrefix: str = SettingsField(title="userAttrPrefix")
+    pythonPerFrameCallback: str = SettingsField(
+        title="Python Per Frame Callback"
+    )
+    pythonPostJobCallback: str = SettingsField(
+        title="Python Post Job Callback"
+    )
+    userAttr: str = SettingsField(title="User Attr")
+    userAttrPrefix: str = SettingsField(title="User Attr Prefix")
```

Comment:
Can we add a `description=""` argument to these explaining the formatting, so that it's clear one needs to use e.g. `,` (comma) separated values. (Or whatever the separator is? maybe it's `;`?)

Reply:
Ah it's `;` according to [this code](https://github.com/ynput/ayon-core/pull/336/files#diff-9ac4195c9cccdcfca6bae8ddc782d74a45ff93e0764f05edd34fff3a98055475R72). Definitely add a description tooltip! Even nicer if we could somehow put in `placeholder` text like `attr1;attr2` and `prefix1;prefix2`. Is that possible in settings?

Reply:
Example:
```suggestion
    userAttr: str = SettingsField(
        title="User Attr",
        placeholder="attr1;attr2",
        description=(
            "Attributes matching by name will be included in the Alembic export. "
            "Attributes should be separated by semi-colon `;`"
        )
    )
    userAttrPrefix: str = SettingsField(
        title="User Attr Prefix",
        placeholder="attr1;attr2",
        description=(
            "Attributes starting with these prefixes will be included in the Alembic export. "
            "Attributes should be separated by semi-colon `;`"
        )
    )
```

Reply:
Same goes for `attr` and `attrPrefix` - and even better if the description could be used to explain the difference between the two. You can include markdown links as well to maybe some Alembic documentation if needed.